### ISSUE TITLE
Add documentation on supported markdown elements

### ISSUE THREAD

**Issue by Henni** [jbt/mdEdit#11]
A few short entries in the wiki about the supported markdown elements would be great.
Possibly with examples.
It's a bit difficult to extract this information from the code because of the partially complex RegExs.

**Comment by jbt** [comment]
Definitely! The idea is that it _should_ just be "everything in markdown" but getting some nice examples would be a good idea
